Index: OgreTerrainSceneManager.h
===================================================================
RCS file: /cvsroot/ogre/ogrenew/PlugIns/OctreeSceneManager/include/OgreTerrainSceneManager.h,v
retrieving revision 1.17
diff -u -3 -r1.17 OgreTerrainSceneManager.h
--- OgreTerrainSceneManager.h	24 Jul 2004 22:30:04 -0000	1.17
+++ OgreTerrainSceneManager.h	30 Jul 2004 19:56:22 -0000
@@ -207,12 +207,12 @@
     /** Returns the TerrainRenderable that contains the given pt.
         If no tile exists at the point, it returns 0;
     */
-    TerrainRenderable * getTerrainTile( const Vector3 & pt );
+    virtual TerrainRenderable * getTerrainTile( const Vector3 & pt );
 
     /** Returns the TerrainPage that contains the given pt.
     If no page exists at the point, it returns 0;
     */
-    TerrainPage* getTerrainPage( const Vector3 & pt );
+    virtual TerrainPage* getTerrainPage( const Vector3 & pt );
 
     /** Creates a RaySceneQuery for this scene manager. 
     @remarks
